velocity/build.gradle

```diff
@@ -1,17 +1,17 @@
 plugins {
     id 'net.kyori.blossom' version '1.3.0'
-    id 'com.github.johnrengelman.shadow' version '7.0.0'
+    id 'com.github.johnrengelman.shadow' version '7.1.2'
 }
 
 repositories {
-    maven { url 'https://repo.velocitypowered.com/releases/' }
+    maven { url 'https://repo.papermc.io/repository/maven-public/' }
 }
 
 dependencies {
     implementation project(':common')
 
-    compileOnly 'com.velocitypowered:velocity-api:3.0.0'
-    annotationProcessor 'com.velocitypowered:velocity-api:3.0.0'
+    compileOnly 'com.velocitypowered:velocity-api:3.1.2-SNAPSHOT'
+    annotationProcessor 'com.velocitypowered:velocity-api:3.1.2-SNAPSHOT'
```

Comment:
Please revert this change

Reply:
You sure you want this reverted as 3.1.2-SNAPSHOT is the current paper version most people use? if you still do, do you want it reverted to 3.0.0 or 3.1.1 as that one is also used more than 3.0.0

Reply:
3.1.1 is fine :)

Reply:
addressed in 0220aa8 :)